src/lib.rs

```diff
@@ -1513,3 +1513,46 @@ macro_rules! impl_tuple_zeroable {
 }
 
 impl_tuple_zeroable!(A, B, C, D, E, F, G, H, I, J);
+
+/// This trait allows creating an instance of `Self` which contains exactly one structurally [`Pinned`](Pin) value.
```

Comment:
```suggestion
/// This trait allows creating an instance of `Self` which contains exactly one [structurally pinned value].
```
Linking to https://doc.rust-lang.org/std/pin/index.html#pinning-is-structural-for-field

Reply:
That anchor is dead

Should it be: https://doc.rust-lang.org/nightly/std/pin/index.html#projections-and-structural-pinning

Reply:
Seems like 1.76 was the last version with these docs:https://doc.rust-lang.org/1.76.0/std/pin/index.html#pinning-is-structural-for-field

From 1.77 onward the anchor is broken.

Reply:
Added a commit that updates the link in the readme.

Reply:
Oh, didn't know they changed the section heading... Thanks for spotting it :)